website/docs/patterns/button-alignment/partials/overview/overview.md

```diff
@@ -0,0 +1,98 @@
+While the alignment, order, and grouping of buttons and actions might seem trivial, the impact on users is immediate and often occurs in experiences of high important and interruption.
+
+These guidelines don't cover every scenario and are intended to aid in the decision-making process when determining how to organize buttons and actions. Different methods of alignment and ordering will result in different outcomes with pros and cons for each. What is most important is **consistency** across an application in how actions are aligned, ordered and grouped.
+
+!!! Info
+
+Button alignment isn't relevant for every component or pattern, but is rather aimed at functional elements; i.e. modals, forms, and page-level actions.
+!!!
+
+## Constrained vs. unconstrained
+
+**Constrained** refers to a component or pattern that is wrapped within a container element, while **unconstrained** refers to components that exist at the page level.
+
+- **Constrained**
+    - Dialogs and modals
+    - Toolbars
+    - Alerts
+    - Banners
+- **Unconstrained**
+    - General page content
+    - In-page forms
+    - The "window" in a general sense
+
+This differentiation is important because the expectations from the user are different in each context. A constrained component or pattern will often be used in an interstitial or circumstantial experience; i.e. an experience that is triggered by the user or the application illiciting a quick, progressive action.
+
+Unconstrained generally refers to a more "static" experience that isn't triggered by a user action. This is often tied to a page-level function or a navigational element.
+
+**Insert some supporting images here**
+
+## Layout and reading patterns
+
+When users read content, the pattern their eyes follow generally adheres to one or more reading pattern. By identifying and familiarizing ourselves  with these patterns we can better understand how the layout within our application, pattern, or component impacts comprehension and interpretation of the content.
+
+The three most common reading patterns are:
+
+- the Gutenberg diagram
+- Z-shaped pattern
+- F-shaped pattern
+
+### Gutenberg diagram
+
+The Gutenberg diagram refers to a pattern the eyes follow when moving or scanning through **evenly distributed, homogenous content.** This pattern is most relevant in text-heavy scenarios like novels and newspapers, but is relevant in a UX context depending on the complexity of the content.
+
+![Visual example of a Gutenberg diagram](/assets/patterns/button-alignment/gutenberg-diagram.png)
+
+#### Characteristics
+
+- The users eyes progress through the content on an **axis of orientation** which moves further away from the start edge on each line or break in the content
+- Content that falls into the weak fallow area recieves less attention
```

Comment:
```suggestion
- Content that falls into the weak fallow area receives less attention
```

Reply:
That's a good point, I don't think it's an often used word, but it's supported by the graphic. That's generally the language that is used when referring to this specific reading pattern.